test/e2e/e2ethanos/custom_test_matchers.go

```diff
@@ -0,0 +1,16 @@
+package e2ethanos
```

Comment:
I can add there in parallel, but I personally do not want to be blocked here until it's merged there. Honestly the flaky tests are being a massive waste of time and resources (paid or not) on my PR, plus we have the penalty dedup test completely skipped since several months.

Reply:
FYI, PR opened upstream: https://github.com/efficientgo/e2e/pull/65